scripts/context.js

```diff
@@ -59,6 +59,7 @@ function singlePageView() {
     chrome.storage.sync.get(['selectedFeature'], function(result) {
       var openedFeature = result.selectedFeature
       let clickFeature = null
+      let countFeature = 0
```

Comment:
Why don't you directly assign it to `features.length ` ?

Reply:
`let features = ['alexa', 'domaintools', 'wbmsummary', 'annotations', 'tagcloud']` this is `features` and it will always return `5` as its length.

Reply:
OK, got it. Thank you